src/ccache.cpp

```diff
@@ -865,124 +865,104 @@ process_preprocessed_file(Context& ctx,
 static void
 use_relative_paths_in_depfile(const Context& ctx)
```

Comment:
Nice, now it's way more understandable what this code is supposed to do :-)

I'll have to assume it's the same as before, hopefully covered by tests.